### ISSUE TITLE
ida 7.3 not found ret-sync plugin

### ISSUE THREAD

**Issue by x55admin** [bootleg/ret-sync#50]
Place the downloaded ida plugin in the plugin directory according to the document method. The plugin cannot be seen in ida, and the version is ida 7.3

**Comment by bootleg** [comment]
Hi @x55admin , 

For Windows setup that would be:

- `C:\Program Files\IDA Pro 7.3\plugins`
- `%APPDATA%\Hex-Rays\IDA Pro\plugins`

I'd recommend the last one though, to have it easier when version upgrades. If the directory does not exists yet, simply create it.

For Linux:
- `~/.idapro/plugins`

**Comment by ktinkone** [comment]
have you slove the question? I have the same question.

**Comment by bootleg** [comment]
@ktinkone 

I don't have more info than above. If you could give me details about what you did or steps to reproduce, it would help me to understand the possible issue here.

Cheers,

**Comment by bootleg** [comment]
Is there still an issue or can we close this one ?

Best regards,

**Comment by bootleg** [comment]
Closing as the issue seems inactive. Please re-open if necessary.

Regards.

**Comment by adrian-rt** [comment]
I have the same problem with IDA 7.2
I don't see any log file being created even though I've enabled logging as above.

Plugin is copied in C:\Users\adria\AppData\Roaming\Hex-Rays\IDA Pro\Plugins

Do I need to have an ext_ida folder in plugins and inside it  retsync folder and SyncPlugin.py? Or I need to have retsync + SyncPlugin.py directly in the plugins folder? anyway I tried both options and nothing seems to work.

Thanks.
